packages/trpc/server/routers/viewer/organizations/create.handler.ts

```diff
@@ -140,6 +153,7 @@ export const createHandler = async ({ input, ctx }: CreateOptions) => {
     seats: seats ?? null,
     pricePerSeat: pricePerSeat ?? null,
     isPlatform,
+    billingPeriod: billingPeriod ? billingPeriod : BillingPeriod.MONTHLY,
```

Comment:
nit: We could define `BillingPeriod.MONTHLY` as default value during the billingPeriod initialisation itself